docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -129,6 +129,10 @@ const theme = createTheme({
 });
 ```
 
+:::warning
```

Comment:
I do think that this should be a success callout, just because it's the info that is added in order to help the user succeed. 🤔 
```suggestion
:::success
```